docker/Dockerfile

```diff
@@ -44,6 +44,8 @@ RUN apt install -y python3-pprintpp; exit 0
 RUN apt clean -y all
 
 RUN update-alternatives --install /usr/bin/python python /usr/bin/python3 10
+RUN mkdir -p `python -m site --user-site`
+ENV PYTHONPATH `python -m site --user-site`
```

Comment:
When I build this image I'm seeing `$PYTHONPATH` equal to "\`python -m site --user-site\`" and not the actual output of that command (confirmed by various [articles](https://forums.docker.com/t/how-do-i-send-runs-output-to-env-in-dockerfile/16106)).  Might need to figure out something more clever to set this dynamically

Reply:
The options I'm seeing are:
1. set it in the `CMD` (don't think this will work with multiple stages since the `RUN` steps will run in different shells
2. store it in a file? (doesn't really solve anything if we need to run a script to set the env var)
3. change the site-packages directory to something we know, so that we know where cyclus is installed and we can safely hardcode the PYTHONPATH (I'm trying to see if this is possible)

Reply:
Actually... do we need to explicitly set the `PYTHONPATH` env var?  When I omit that line and build images, the correct site-packages directory is in `sys.path` and all tests pass (besides test_main.py of course). The best solution might be to delete line 48 altogether